### ISSUE TITLE
Browserify: response.setEncoding is not a function

### ISSUE THREAD

**Issue by davidebaldini** [tmpvar/jsdom#1019]
The procedure of this failing test is the same as of #1018 (thank you for helping with that):
* I browserify jsdom standalone, without a "[wrapper] (https://github.com/tmpvar/jsdom/issues/1018#issuecomment-73122321)" project.
* the HTML component is the same as of #1018, except for the addition of the `scripts` property:
```javascript
<script src='/src/lib/jsdomBrowserify.js'></script>
<script>
    jsdom = require('jsdom');
    jsdom.env({
        html:  '<div></div>',
        scripts: ['http://example.net/d3.js'],
        done: function (errors, window) {
            console.log('message');
        }
    });
</script>
```

I load the `d3` library, among other simpler ones.
The browser's error console reports:
"*TypeError: response.setEncoding is not a function*", file:  jsdomBrowserify.js:26867

**Comment by Sebmaster** [comment]
Is the script a cross-origin request?

**Comment by davidebaldini** [comment]
No, the main HTML page and all the `scripts` array elements source from the same host / subdomain, and same HTTP protocol.

**Comment by davidebaldini** [comment]
If it may be useful, I stripped down a minimal HTML snipped able to reproduce the error.
Also served live at http://www.hwcharts.com/jsdomTest0.html
```html
<!doctype html>
<html><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8" />

<script src='/src/lib/jsdomBrowserify.js'></script>
<script>
    jsdom = require('jsdom');
    jsdom.env({
        html:  '<div id=cloud-tag></div>',
        scripts: ['http://www.hwcharts.com/src/lib/d3/d3.js.sh',
                  'http://www.hwcharts.com/src/lib/d3-cloud/d3.layout.cloud.js.sh',
                  'http://www.hwcharts.com/src/lib/cloudGenerator.js.sh'],
        done: function (errors, window) {
            console.log();
        }
    });
</script>
</head><body>
</body></html>
```

**Comment by domenic** [comment]
OK, I fixed *this* error with 1aec278d12da29ed9bdf1b65c9c8994ba9c57533. However, we still can't get the `scripts:` entry working, because we don't have the ability to run scripts inside a specific context inside browsers. So I'll close this and open a new one.

**Comment by MadLittleMods** [comment]
:+1: for solution. Also a problem with `webpack` but this fix solved it in [another package](https://github.com/gitterHQ/node-gitter/issues/16) :grinning:
